python/tvm/hybrid/runtime.py

```diff
@@ -86,6 +86,37 @@ def sigmoid(x):
     'tanh'         : numpy.tanh,
     'power'        : numpy.power,
     'exp'          : numpy.exp,
+    'uint8'        : numpy.uint8,
+    'uint16'       : numpy.uint16,
+    'uint32'       : numpy.uint32,
+    'uint64'       : numpy.uint64,
+    'int8'         : numpy.int8,
+    'int16'        : numpy.int16,
+    'int32'        : numpy.int32,
+    'int64'        : numpy.int64,
+    'float16'      : numpy.float16,
+    'float32'      : numpy.float32,
+    'float64'      : numpy.float64,
     'sigmoid'      : sigmoid,
```

Comment:
Could you also add `ceil` here? would be an easy one

Reply:
Nice suggestion!